documentation/Troubleshooting.md

```diff
@@ -927,6 +930,38 @@ spec:
         - '--disable-full-test'
 ```
 
+## CoreDNS Cannot Resolve the Name
+
+### Case 1
+
+**Symptoms**: A pod can't resolve the non-FQDN. Check inside the pod looks the following:
+
+```
+$ nslookup kubernetes.default
+Server:         172.30.0.10
+Address:        172.30.0.10:53
+
+
+** server can't find kubernetes.default: NXDOMAIN
+```
+
+**Root cause**: Images that are based on Alpine Linux do not use `search` directives in `/etc/resolv.conf` by default.
```

Comment:
It's not Alpine Linux image per se, it's busybox utility in the Alpine, who causes this error (for example, see https://github.com/docker-library/busybox/issues/48), so that may happen with other images too.

Reply:
Could you give an example, please?
I'm going to add WA from: [Does Alpine have known DNS issue within Kubernetes?](https://stackoverflow.com/questions/65181012/does-alpine-have-known-dns-issue-within-kubernetes)
Maybe there is something else that I should mention

Reply:
Yes, this WA is fine for Alpine-based images.
Busybox images themselves also can have issues, for example, versions <=1.28.4 and >=1.35.0 resolve short names, but intermediate versions (for example, 1.29.3 or 1.33) give errors for existing names.

Reply:
Thanks, I'll take it into account